extensions/arc/src/types.ts

```diff
@@ -38,3 +38,7 @@ export type Suggestion = {
   query: string;
   url: string;
 };
+
+export type Arcguments = {
```

Comment:
Probably best to call this something like `NewLittleArcArguments` to make it more explicit but can be done with the next codechange.